platforms/android/AndroidManifest.xml

```diff
@@ -33,14 +29,6 @@
             <meta-data android:name="android.accounts.AccountAuthenticator" android:resource="@xml/authenticator" />
         </service>
         <service android:name="com.mb.android.media.RemotePlayerService" />
-        <service android:name="com.mb.android.media.legacy.KitKatMediaService" />
-        <service android:name="com.mb.android.media.PlaybackService" />
-        <service android:name="com.mb.android.media.MediaService" />
-        <receiver android:name="com.mb.android.media.legacy.RemoteControlClientReceiver">
-            <intent-filter>
-                <action android:name="android.intent.action.MEDIA_BUTTON" />
```

Comment:
care to explain why it's removed?

Reply:
I removed all files related to VLC playback, and this was one of them.

Reply:
So now Jellyfin won't react to play/pause button, or am I reading this wrong?

Reply:
No it still works fine, this was a separate media player that is currently unused.